iDiff/differs/pipDiff.go

```diff
@@ -64,18 +66,28 @@ func (d PipDiffer) getPackages(path string) (map[string]utils.PackageInfo, error
 			if len(packageMatch) != 0 {
 				packageName := packageMatch[1]
 				version := packageMatch[2][:len(packageMatch[2])-1]
-				size := strconv.FormatInt(c.Size(), 10)
-				packages[packageName] = utils.PackageInfo{version, size}
 
-				continue
-			}
+				// Retrieves size for actual package/script corresponding to each dist-info metadata directory
+				// by taking the file entry alphabetically before it (for a package) or after it (for a script)
```

Comment:
I'm a little confused by this, can you clarify? Why do we want to look at the previous file entry to find the size?